<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageWidget</class>
 <widget class="QWidget" name="ManageWidget">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
color:rgb(78, 78, 78，80);
background-color: rgb(247, 235, 183);
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="peopleComboBox">
     <property name="styleSheet">
      <string notr="true">
QComboBox:!hover{
	background-color: rgb(255, 255, 255,30);
	font: 14pt &quot;仿宋&quot;;
	color: rgb(0, 0, 0);
        border-width:2px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 2px 2px;
    }
QComboBox:hover{
	background-color: rgb(255, 255, 255,90);
	font: 14pt &quot;仿宋&quot;;
	color: rgb(0, 0, 0);
        border-width:2px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 2px 2px;
    }</string>
     </property>
     <item>
      <property name="text">
       <string>请选择人员</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="informationLabel">
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;仿宋&quot;;</string>
     </property>
     <property name="text">
      <string>人员信息</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="levelComboBox">
     <property name="styleSheet">
      <string notr="true">
QComboBox:!hover{
	background-color: rgb(255, 255, 255,30);
	font: 14pt &quot;仿宋&quot;;
	color: rgb(0, 0, 0);
        border-width:2px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 2px 2px;
    }
QComboBox:hover{
	background-color: rgb(255, 255, 255,90);
	font: 14pt &quot;仿宋&quot;;
	color: rgb(0, 0, 0);
        border-width:2px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 2px 2px;
    }</string>
     </property>
     <item>
      <property name="text">
       <string>权限等级</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="changeButton">
     <property name="styleSheet">
      <string notr="true">QPushButton:!hover{
	background-color: rgb(123, 117, 69,30);
	font: 16pt &quot;仿宋&quot;;
	color: rgb(0, 0, 0);
        border-width:2px;
        border-style:solid;
        border-color:gray;
        border-radius:10px;
        padding: 2px 2px;
    }
QPushButton:hover{
	background-color: rgb(123, 117, 69,90);
	font: 16pt &quot;仿宋&quot;;
	color: rgb(0, 0, 0);
        border-width:2px;
        border-style:solid;
        border-color:gray;
        border-radius:10px;
        padding: 2px 2px;
    }</string>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
